msal-web-sample/src/main/java/com/microsoft/azure/msalwebsample/StateData.java

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.microsoft.azure.msalwebsample;
+
+import lombok.Getter;
+
+import java.util.Date;
+
+@Getter
```

Comment:
I realize that you didn't introduce this, but can we remove use of lombok in the sample? It requires a plugin for IDEs which was causing people trouble who weren't familiar with it so we decided to stop using it in samples. As far as I can tell, it's only used here (should also be deleted from the pom.xml)

Reply:
Just removed Lombok and it's references, and replaced it with all of the needed getters/setters.